docs/getting-started/nullblk.md

```diff
@@ -81,9 +81,17 @@ command to configure the emulated disk.
 	zone_size=64 \
 ```
 
+In this example, the arguments mean the following:
+
+   1. ``nr_devices=1`` means that the number of specified devices is one (1). 
```

Comment:
May be: "...that only one device will be created" or "that the number of devices to create is one" ?
Since we do not "specify" any device here, I find the "specified devices" wording confusing...

Reply:
I think you're right.

Reply:
I've made this change.